packages/css/src/dialog/README.md

```diff
@@ -0,0 +1,22 @@
+# Dialog
+
+The dialog provides feedback or reveals additional information or functionality while maintaining the context of an underlying page.
+
+## Guidelines
+
+Use modals sparingly because they interrupt the user's workflow.
```

Comment:
```suggestion
- Use Dialog sparingly, because it interrupts the user's workflow.
- Use Dialog for short and non-frequent tasks. For common tasks consider using the main flow.
```